Makefile

```diff
@@ -335,7 +335,6 @@ LIBC_TOP_HALF_MUSL_SOURCES += \
         thread/sem_timedwait.c \
         thread/sem_trywait.c \
         thread/sem_wait.c \
-        thread/$(TARGET_ARCH)/wasi_thread_start.s \
```

Comment:
We put this file back in before, it must have been removed by mistake here.

Reply:
I move this to a separate PR #69